tools/search_quran.py

```diff
@@ -47,8 +47,12 @@ def run(self, query: str, num_results: int=5):
     
     def pp_ayah(self, ayah):
         ayah_num = ayah['id']
-        ayah_ar = ayah['text']
-        ayah_en = ayah['en_text']
+        ayah_ar = "Not retrieved"
```

Comment:
Did you find instances where this was happening or is this defensive programming?

Reply:
Yes, it does indeed happen. Please refer to this issue for an example: [this issue](https://github.com/waleedkadous/ansari-backend/issues/20).